lib/streamlit/util.py

```diff
@@ -103,10 +106,22 @@ def _maybe_tuple_to_list(item: Any) -> Any:
     return item
 
 
-def repr_(cls) -> str:
-    classname = cls.__class__.__name__
-    args = ", ".join([f"{k}={repr(v)}" for (k, v) in cls.__dict__.items()])
-    return f"{classname}({args})"
+def repr_(self: Any) -> str:
```

Comment:
Are we comfortable with hiding probably-default values in reprs of various regular classes? Do other people want a different set of values excluded?